specification/storagesync/resource-manager/Microsoft.StorageSync/preview/2020-03-01/storagesync.json

```diff
@@ -162,13 +162,39 @@
               "$ref": "#/definitions/StorageSyncService"
             }
           },
+          "202": {
```

Comment:
Continue from previous PR.

This appears not consistent with most common services.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json#L915-L934